tasks/Module 4/task_3.py

```diff
@@ -0,0 +1,19 @@
+"""Run Selective Copy."""
+import os
+
+
+def selective_copy(source_folder, destination_folder, extention):
+    """Run Selective copy."""
+    #os.chdir(source_folder)
+    os.makedirs(destination_folder, exist_ok=True)
+    with os.scandir(source_folder) as scandir:
```

Comment:
Please notice that task description says "Write a program that walks through a **folder tree** and...". Maybe it is not clearly stated, but I believe that idea was to include subdirectories as well. You can use `os.walk` to do this.

Reply:
fixed